metadata/src/lib.rs

```diff
@@ -93,6 +93,14 @@ pub struct Album {
     pub covers: Vec<FileId>,
 }
 
+#[derive(Debug, Clone)]
+pub struct Playlist {
```

Comment:
Should include an `id` field like the others do. Exposing the playlist `revision` might also be quite useful.

Reply:
I'll add that and update the PR!

Reply:
Hmm the id isn't actually included in the message. It's included for all the others, just playlists don't have it. I don't know if there is any way to get the id, except just reusing the id from the request.
Revision is a 24 element long array of u8. Ok to just use a Vec<u8>?

Reply:
Have been thinking a bit more about this. Reusing the id from the request gets too ugly. And I haven't figured out any other way to get it. I had a small hope that it would be included as part of the (really long) revision, but unfortunately no. Can't we just leave it out for the time being?